docs/deployment.md

```diff
@@ -215,4 +215,4 @@ Probot comes bundled with a client for the [Sentry](https://github.com/getsentry
   3. Set the `SENTRY_DSN` environment variable with the DSN you retrieved.
 
 ## Serverless
-Serverless abstracts away the most menial parts of building an application, leaving developers to write code and not actively manage scaling for their applications. The [Serverless Deployment](/serverless-deployment) section will show you how to deploy you application using functions instead of servers.
+Serverless abstracts away the most menial parts of building an application, leaving developers to write code and not actively manage scaling for their applications. The [Serverless Deployment](https://probot.github.io/docs/serverless-deployment) section will show you how to deploy you application using functions instead of servers.
```

Comment:
```suggestion
Serverless abstracts away the most menial parts of building an application, leaving developers to write code and not actively manage scaling for their applications. The [Serverless Deployment](./serverless-deployment) section will show you how to deploy you application using functions instead of servers.
```

Reply:
I totally agree. 

The reason I used a full URL is because that's what's used in the majority of cases:
**Relative references**
![image](https://user-images.githubusercontent.com/4658208/48731453-0c4d7b80-ec0b-11e8-81e7-aa0e02fd3254.png)

**Absolute references**
![image](https://user-images.githubusercontent.com/4658208/48731468-17a0a700-ec0b-11e8-8f13-d90a22e3912a.png)

If you would prefer relative references, I can go ahead and update all other absolute references as part of this PR.

Reply:
@matchai thanks for pointing that out, that would be great!